code/modules/clothing/under/accessories/accessory.dm

```diff
@@ -448,6 +448,7 @@
 	else
 		emagged = TRUE
 		to_chat(user, "<span class='warning'>You swipe the card and crack the holobadge security checks.</span>")
```

Comment:
Also just noting that many of the emag descriptions describe the card-swiping, which clearly won't be the case when you're quickhacking these things lol

Reply:
discussed on discord, not worrying about in this pr as revenants / pulse demons do the same thing